device/ipv4.yaml

```diff
@@ -57,7 +59,23 @@ components:
           type: string
           format: ipv4
           default: 0.0.0.0
-        
+
+    Device.Ipv4ManualGatewayMAC:
+      description: >- 
+        Object to set Manual Gateway MAC
+      type: object
+      properties:
+        enable:
+          description: >-
+            By default Manual Gateway MAC is not enabled and the resolved Gateway is used.
+            To overwrite the resolved Gateway with Manual Gateway(address), set enable to True.
+          type: boolean
+          default: false
+        address:
+          description: >-
+            Manual Gateway MAC.
```

Comment:
"When enable_configured_gw_mac is set to true, this user-configured MAC address will be used  for auto-filling the destination MAC address in the control and data packets sent from this IPv4 endpoint  whenever applicable. This MAC must match the MAC of the interface on the connected device where the gateway_ip is configured. " -> Something like this might be more accurate maybe ?

Reply:
Note : I am not repeating the similar comments for IPv6. Changes if done should be done symmetrically in both places.

Reply:
updated